<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sceneRect">
        <rectf>
         <x>0.000000000000000</x>
         <y>0.000000000000000</y>
         <width>20.000000000000000</width>
         <height>20.000000000000000</height>
        </rectf>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGraphicsView" name="graphicsView_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="graphicsView_4">
       <property name="sceneRect">
        <rectf>
         <x>3.000000000000000</x>
         <y>3.000000000000000</y>
         <width>5.000000000000000</width>
         <height>5.000000000000000</height>
        </rectf>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGraphicsView" name="graphicsView_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
